sample-app/screens/Home/HomeData.tsx

```diff
@@ -128,7 +128,7 @@ const parsePaymentMethods = (values: Array<String>): JudoPaymentMethod => {
   return paymentMethods
 }
 
-const parseCardNetworks = (values: Array<String>): JudoCardNetwork => {
+const parseCardNetworks = (values: [string]): JudoCardNetwork => {
```

Comment:
isn't it supposed to be `string[]`?
Setting `[string]` means having an array with only 1 string element inside.

Reply:
👍